### ISSUE TITLE
Libraries incompatible with Node 18

### ISSUE THREAD

**Issue by akshaybabloo** [TryGhost/Ghost#16059]
### Issue Summary

`kg-default-cards` errors on Node 18

### Steps to Reproduce

1. Install Node 18
2. Try to upgrade your current ghost version

### Ghost Version

5.26.3

### Node.js Version

18.12.1

### How did you install Ghost?

On Windows locally

### Database type

SQLite3

### Relevant log / error output

```shell
Debug Information:
    OS: Microsoft Windows 10 Pro, v10.0.19044
    Node Version: v18.12.1
    Ghost Version: 5.2.3
    Ghost-CLI Version: 1.24.0
    Environment: development
    Command: 'ghost update'
Message: Command failed: yarn install --no-emoji --no-progress
warning ghost@5.26.4: The engine "cli" appears to be invalid.
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.27.0"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@^0.5.31"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.4"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.4"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@0.5.34"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.3"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@^0.5.33"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.27.0"
warning Resolution field "@tryghost/logging@2.3.5" is incompatible with requested version "@tryghost/logging@2.2.3"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.29.1"
error @tryghost/kg-default-cards@5.18.7: The engine "node" is incompatible with this module. Expected version "^12.22.1 || ^14.17.0 || ^16.13.0". Got "18.12.1"
error Found incompatible module.

yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

Exit code: 1

--------------- stdout ---------------
yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.


--------------- stderr ---------------
warning ghost@5.26.4: The engine "cli" appears to be invalid.
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.27.0"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@^0.5.31"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.4"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.4"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@0.5.34"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.1"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.29.3"
warning Resolution field "moment-timezone@0.5.23" is incompatible with requested version "moment-timezone@^0.5.33"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@2.27.0"
warning Resolution field "@tryghost/logging@2.3.5" is incompatible with requested version "@tryghost/logging@2.2.3"
warning Resolution field "moment@2.24.0" is incompatible with requested version "moment@^2.29.1"
error @tryghost/kg-default-cards@5.18.7: The engine "node" is incompatible with this module. Expected version "^12.22.1 || ^14.17.0 || ^16.13.0". Got "18.12.1"
error Found incompatible module.
```

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by daniellockyer** [comment]
Hi @akshaybabloo 🙂 The latest version of Ghost doesn't actually support Node 18 yet, but a couple of PRs have been prematurely merged that would allow Ghost-CLI to assume it does.

I've just merged support for Node 18 into `main` so it'll be out in the next release, which should be soon. I'll update here once it's released

**Comment by ErisDS** [comment]
I'm sure you've caught this, but the error is nothing to do with moment:

> error @tryghost/kg-default-cards@5.18.7: The engine "node" is incompatible with this module. Expected version "^12.22.1 || ^14.17.0 || ^16.13.0". Got "18.12.1"
error Found incompatible module.

**Comment by akshaybabloo** [comment]
@ErisDS Yes, true. That's why I have edited the content and the title a while ago.

**Comment by daniellockyer** [comment]
This should now be fixed in Ghost 5.27.0 😀

**Comment by zznobzz** [comment]
The problem is actual.
![image](https://user-images.githubusercontent.com/23694034/223801359-530eefe5-beab-4a9a-9d75-b8153ae0e692.png)
